src/dev/start_dev_server.js

```diff
@@ -261,6 +261,7 @@ export const startDevServer = async ({
             nodeEsmResolution,
             magicExtensions,
             magicDirectoryIndex,
+            directoryReferenceEffect: "preserve",
```

Comment:
Allow dev server to display directory content, before it would have thrown an internal 500 error.